src/aere_utils.erl

```diff
@@ -0,0 +1,46 @@
+-module(aere_utils).
```

Comment:
`aere_files`? It only does file stuff, no need to call it utils I think?

Reply:
Changed here 88f1c40eca14d7a9de43ef291f39b33b324c877f.